### PR TITLE
[BRND-222] API docs rebranding

### PR DESCRIPTION
**Pull request by aiyer-smartsheet**
As part of rebranding, renaming all the references to 10,000 to Resource Management by Smartsheet.

### CODE REVIEW

**Review by rchaudhry-smartsheet — APPROVED** [review]

**Review by baonguyen84 — APPROVED** [review]
LGTM

File: source/includes/_zapier-integration.md [hunk 1]
```diff
@@ -116,13 +116,13 @@ Familiarize yourself with the different types of users and assignables available
...
 
-## More details about the 10,000ft API
+## More details about the Resource Management by Smartsheet API
 
-Zapier is a convenient and powerful way to consume the 10,000ft API. The triggers, actions and searches listed above allow you to access commonly used API functionality without having to develop custom API integrations or write software code.
+Zapier is a convenient and powerful way to consume the Resource Management by Smartsheet API. The triggers, actions and searches listed above allow you to access commonly used API functionality without having to develop custom API integrations or write software code.
 
-The 10,000ft API provides access to a number of advanced features that are not currently supported with our Zapier triggers and actions. See [10,000ft API documentation](#10,000ft-api-(pro-and-enterprise-only)) for more details.
+The Resource Management by Smartsheet API provides access to a number of advanced features that are not currently supported with our Zapier triggers and actions. See [Resource Management by Smartsheet API documentation](#Resource Management by Smartsheet-api-(pro-and-enterprise-only)) for more details.
```
Comment (rchaudhry-smartsheet) [inline source/includes/_zapier-integration.md]: We should probably check the # part. Is that supposed to link to another part of the docs?

Comment (aiyer-smartsheet) [reply source/includes/_zapier-integration.md]: Good catch, missed this with the bulk renaming. This should be fixed now.

**Review by rchaudhry-smartsheet — APPROVED** [review]
lgtm

File: source/index.md.erb [hunk 2]
```diff
@@ -106,30 +106,30 @@ The API provides access to the following data collections in your account via RE
...
-A few of our customers have implemented and shared API client libraries to help using the 10Kft API easier. Depending on your scenario, you may benefit from adopting and improving on them. Here are a couple of references,
+A few of our customers have implemented and shared API client libraries to help using the Resource Management by Smartsheet API easier. Depending on your scenario, you may benefit from adopting and improving on them. Here are a couple of references,
 
 * Revelry Labs / `ruby` / [https://github.com/revelrylabs/tenk](https://github.com/revelrylabs/tenk)
 * Spindance / `ruby` / [https://github.com/spindance/ten_thousand_feet](https://github.com/spindance/ten_thousand_feet)
 
 # Questions?
 
-Questions or feedback about our API, please submit a support ticket at our [Support Page](https://help.smartsheet.com/contact/10000ft)
+Questions or feedback about our API, please submit a support ticket at our [Support Page](https://help.smartsheet.com/contact/resource-management)
```
Comment (baonguyen84) [inline source/index.md.erb:133]: Do we need to do anything here or other team need to change it? The new url are not be able to reach @aiyer-smartsheet

Comment (aiyer-smartsheet) [reply source/index.md.erb:133]: There is a team working on enabling the new Resource management pages now. We should have this live by eod.

File: source/index.md.erb [hunk 3]
```diff
@@ -72,7 +72,7 @@ API access is available on Pro, Enterprise and Business plans of 10000ft. As of
 
 # Need help?
 
-* Contact us via our [Support Page](https://help.smartsheet.com/contact/10000ft)
+* Contact us via our [Support Page](https://help.smartsheet.com/contact/resource-management)
```
Comment (baonguyen84) [inline source/index.md.erb:75]: Same with below question and others across the changes, related to support page.

**Comment by baonguyen84** [comment]
Look like we do not re-approve. Thank for answering the question @aiyer-smartsheet ✅

File: source/index.md.erb (hunk 2 above)
Comment (rchaudhry-smartsheet) [reply source/index.md.erb:133]: @baonguyen84 This link will be working soon. There is a team getting it up. It's the same link in prod.

**Pull request merged**